### PR TITLE
Adds Equipment Campaign feature

### PR DESCRIPTION
**Pull request by ahadziii**

### CODE REVIEW

**Comment by jaedag** [comment]
So is this the pR with the final changes?

**Comment by dabick14** [comment]
> So is this the pR with the final changes? @dabick14  @ahadziii 
> 
> 

I think so

**Review by jaedag — APPROVED** [review]

**Comment by ahadziii** [comment]
I want to add the ApolloWrapper before it is merged

File: api/src/schema/campaigns-equipment.graphql [hunk 1]
```diff
@@ -9,8 +9,8 @@ interface EquipmentRecord {
 }
 
 type ConstituencyEquipment implements EquipmentRecord {
-  id: ID!
-  pulpits: Int!
+  id: ID
+  pulpits: Int
```
Comment (jaedag) [inline api/src/schema/campaigns-equipment.graphql:13]: Why would rather take the exclamation marks off?

File: web-react-ts/src/pages/campaigns/equipment/gathering-service/GatheringServiceEquipmentCamPaign.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react'
+import { Container } from 'react-bootstrap'
+import MenuButton from '../../components/buttons/MenuButton'
+import { useNavigate } from 'react-router'
+import { MemberContext } from 'contexts/MemberContext'
```
Comment (jaedag) [inline web-react-ts/src/pages/campaigns/equipment/gathering-service/GatheringServiceEquipmentCamPaign.tsx:5]: This is a strange file name

**Review by jaedag — APPROVED** [review]

File: api/src/schema/campaigns-equipment.graphql (hunk 1 above)
Comment (dabick14) [reply api/src/schema/campaigns-equipment.graphql:13]: When the fellowships have filled their forms, but the constituency has not, there will be some data. Making the ID and pulpits non nullable will throw an error even though there is data for the constituency

**Review by dabick14 — APPROVED** [review]

File: web-react-ts/src/pages/campaigns/equipment/gathering-service/GatheringServiceEquipmentCamPaign.tsx (hunk 2 above)
Comment (ahadziii) [reply web-react-ts/src/pages/campaigns/equipment/gathering-service/GatheringServiceEquipmentCamPaign.tsx:5]: I am fixing that now

**Pull request merged**